WebApp/src/main/java/com/isa/webapp/model/GradeForm.java

```diff
@@ -1,57 +1,15 @@
 package com.isa.webapp.model;
 
+import lombok.Getter;
+import lombok.Setter;
 
-import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.ObjectMapper;
-
-import java.io.File;
-import java.io.IOException;
-import java.util.List;
-
+@Getter
+@Setter
 public class GradeForm {
```

Comment:
to można wydzielić do pakietu `dto`

Reply:
do pakietu `dto`, a nazwa `GradeForm` była okej, bo obiekt służy tylko do zebrania info z frontu, ale nadal obiekt służy jako dto, czyli data transfer object